circle.yml

```diff
@@ -18,4 +18,5 @@ dependencies:
 
 test:
   override:
-    - scripts/run_test_suite.sh
+    - scripts/run_test_suite.sh:
+        timeout: 3600
```

Comment:
Circle has a maximum timeout of 2 hrs, I'll give it another 30 minutes on the next push.